memlab.bib

```diff
@@ -19365,6 +19540,223 @@ @article{TroyEtal97
 	Volume = {11},
 	Year = {1997}}
 
+@article{KahnBeat66,
+  Title = {{Pupil diameter and load on memory}},
+  Year = {1966},
+  Journal = {Science},
+  Author = {Kahneman, D. and Beatty, J.,
+  Number = {3756},
+  Pages = {1583-1585},
+  Volume = {154},
+  Doi = {10.1126/science.154.3756.1583},
+}
+
+@article{KahnEtal67,
+  Title = {{Perceptual deficit during a mental task}},
+  Year = {1967},
+  Journal = {Science},
+  Author = {Kahneman, D. and Beatty, J. and Pollack, I.},
+  Number = {3785},
+  Pages = {218-219},
+  Volume = {157},
+  Publisher = {American Association for the Advancement of Science},
+}
+
+@article{KrueTian04,
+  Title = {{A comparison of the general linear mixed model and repeated measures ANOVA using a dataset with multiple missing data points}},
```

Comment:
for this title it should be `{A comparison of the general linear mixed model and repeated measures {ANOVA} using a dataset with multiple missing data points}`